src/display/editor/tools.js

```diff
@@ -1109,16 +1113,15 @@ class AnnotationEditorUIManager {
   }
 
   #addSelectionListener() {
-    document.addEventListener("selectionchange", this.#boundSelectionChange);
-  }
-
-  #removeSelectionListener() {
-    document.removeEventListener("selectionchange", this.#boundSelectionChange);
+    document.addEventListener("selectionchange", this.#boundSelectionChange, {
```

Comment:
Could we bind the listener function inline, since we no longer need to remove it manually?
```suggestion
    document.addEventListener("selectionchange", this.#selectionChange.bind(this), {
```